### PR TITLE
Added Signup option in navigation bar.

### PR DESCRIPTION
**Pull request by plxity**
Fixes #2010 

Changes: Added Sign Up option in Top Bar

Demo Link: https://pr-2012-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
capture

### CODE REVIEW

**Comment by plxity** [comment]
@anshumanv @akshatnitd Please review this. :)

File: src/components/ChatApp/TopBar.react.js [hunk 1]
```diff
@@ -256,6 +259,16 @@ class TopBar extends Component {
               onTouchTap={toggleShareClose}
               rightIcon={<Share />}
             />
+            {accessToken ? (
```
Comment (akshatnitd) [inline src/components/ChatApp/TopBar.react.js]: Use `&&` inplace of ternary here

**Comment by anshumanv** [comment]
Not sure if we need this as this will break the consistency across all apps, so we gotta do it everywhere in case we decide to do it here.

**Comment by plxity** [comment]
@anshumanv I feel adding this option will provide smooth functioning for new users to signup. and in https://accounts.susi.ai/ I think we don't need this as homepage already has an option for signup. We need to add this only in https://skills.susi.ai/. If you approve this, I will create a similar issue in other repo.

File: src/components/ChatApp/TopBar.react.js (hunk 1 above)
Comment (plxity) [reply src/components/ChatApp/TopBar.react.js]: Okay, sir.

**Comment by plxity** [comment]
@anshumanv What should I do?

**Comment by anshumanv** [comment]
@akshatnitd what do you think about the change? 🤔

**Comment by plxity** [comment]
@fragm3 but why?

**Comment by plxity** [comment]
@akshatnitd What do you suggest or think about this feature? :)

**Comment by plxity** [comment]
@akshatmitd Okay, but if someone is new to this application and wants to sign-up, only "Login" option will appear in the navigation bar. It will be confusing. But if Sign-up option is present it will provide a direct link to Sign-up.
That's all from my side :)

**Comment by plxity** [comment]
@mariobehling Sir, What do you think about this feature? :)

**Comment by plxity** [comment]
@mariobehling Sir, What are your views on this?

**Comment by fragm3** [comment]
Not required, closing.

**Pull request closed without merging**